src/backend/distributed/commands/type.c

```diff
@@ -129,31 +130,13 @@ PreprocessCompositeTypeStmt(Node *node, const char *queryString,
 	 */
 	EnsureCoordinator();
 
-	/* fully qualify before lookup and later deparsing */
-	QualifyTreeNode(node);
-
-	/*
-	 * reconstruct creation statement in a portable fashion. The create_or_replace helper
-	 * function will be used to create the type in an idempotent manner on the workers.
-	 *
-	 * Types could exist on the worker prior to being created on the coordinator when the
-	 * type previously has been attempted to be created in a transaction which did not
-	 * commit on the coordinator.
-	 */
-	const char *compositeTypeStmtSql = DeparseCompositeTypeStmt(node);
-	compositeTypeStmtSql = WrapCreateOrReplace(compositeTypeStmtSql);
-
 	/*
 	 * when we allow propagation within a transaction block we should make sure to only
 	 * allow this in sequential mode
 	 */
 	EnsureSequentialMode(OBJECT_TYPE);
```

Comment:
I suppose we could move this along with NodeDDLTaskList, since we don't really need to be in sequential mode if we don't propagate anything.